cmd/etl_worker/etl_worker.go

```diff
@@ -197,7 +197,12 @@ func worker(w http.ResponseWriter, r *http.Request) {
 	p := parser.NewParser(dataType, ins)
 	tsk := task.NewTask(fn, tr, p)
 
-	err = tsk.ProcessAllTests()
+	files, err := tsk.ProcessAllTests()
+
+	// Count the files processed per-host-module per-weekday.
+	metrics.FileCount.WithLabelValues(
+		data.Host+"-"+data.Pod+"-"+data.Experiment,
```

Comment:
Good idea. I'd like to see how the current version works in practice. I'm still skeptical about the entire approach of joining daily metrics with prometheus. If the single-metric approach is indeed helpful then I'll look at separate labels here to gain the additional flexibility. Added a TODO.